test-data/local/snake_oil_structure/ert/input/observations/obsfiles/observations.txt

```diff
@@ -4,53 +4,47 @@ SUMMARY_OBSERVATION WOPR_OP1_9
 {
     VALUE   = 0.1;
     ERROR   = 0.05;
-    RESTART = 9;
+    DATE    = 2010-03-31;  -- (RESTART = 9)
     KEY     = WOPR:OP1;
 };
 
 SUMMARY_OBSERVATION WOPR_OP1_36
 {
     VALUE   = 0.7;
     ERROR   = 0.07;
-    RESTART = 36;
+    DATE    = 2010-12-26;  -- (RESTART = 36)
     KEY     = WOPR:OP1;
 };
 
 SUMMARY_OBSERVATION WOPR_OP1_72
 {
     VALUE   = 0.5;
     ERROR   = 0.05;
-    RESTART = 72;
+    DATE    = 2011-12-21;  -- (RESTART = 72)
     KEY     = WOPR:OP1;
 };
 
 SUMMARY_OBSERVATION WOPR_OP1_108
 {
     VALUE   = 0.3;
     ERROR   = 0.075;
-    RESTART = 108;
+    DATE    = 2012-12-15; -- (RESTART = 108)
     KEY     = WOPR:OP1;
 };
 
 SUMMARY_OBSERVATION WOPR_OP1_144
 {
     VALUE   = 0.2;
     ERROR   = 0.035;
-    RESTART = 144;
+    DATE    = 2013-12-10;  -- (RESTART = 144)
     KEY     = WOPR:OP1;
 };
 
 SUMMARY_OBSERVATION WOPR_OP1_190
 {
     VALUE   = 0.015;
     ERROR   = 0.01;
-    RESTART = 190;
+    DATE    = 2015-03-15;  -- (RESTART = 190)
     KEY     = WOPR:OP1;
 };
 
-GENERAL_OBSERVATION WPR_DIFF_1 {
```

Comment:
Note to reviewer: This is a bugfix, as this observation is invalid and causes error message for the snake_oil_structure config.